CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ All notable changes to this project will be documented in this file.
 ### New
 - GraphQL optimization: use single web socket to serve all subscriptions.
 - `net.find_last_shard_block` function returning account shard last block ID.
+- **Debot Module:**
+  - Add new browser callback `switch_completed` to notify browser when all context actions are shown.
```

Comment:
Mention new enum variant for this callback

Reply:
Fixed